### PR TITLE
Feature grow 634 loan description

### PR DESCRIPTION
**Pull request by mlibazisi**
Ticket: GROW-634

### CODE REVIEW

File: src/components/BorrowerProfile/LoanDescription.vue [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<template>
+	<section>
+		<p
+			class="tw-text-h2"
+		>
+			More about this loan
```
Comment (BoulderBrains) [inline src/components/BorrowerProfile/LoanDescription.vue:6]: Should this be "{{ borrowerName }}'s story"?

Comment (mlibazisi) [reply src/components/BorrowerProfile/LoanDescription.vue:6]: You're right, I was looking at the older sigma file. Will update!

File: src/components/BorrowerProfile/LoanDescription.vue [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+		loanAlertTextParagraphs() {
+			return this.toParagraphs(this.loanAlertText);
+		},
+		businessDescriptionParagraphs() {
+			return this.toParagraphs(this.businessDescription);
+		},
+		purposeParagraphs() {
+			return this.toParagraphs(this.purpose);
+		},
+		toParagraphs(text) {
```
Comment (emuvente) [inline src/components/BorrowerProfile/LoanDescription.vue:116]: This should go in `methods` instead of `computed`

**Review by emuvente — COMMENTED** [review]
I think the `<p>` tags that have `text-h2` and `text-h3` classes could actually be changed into `<h2>` and `<h3>` tags, as they are actually headings instead of paragraphs.

Other than that, this looks good!

**Review by BoulderBrains — APPROVED** [review]

**Pull request merged**